.travis.yml

```
@@ -26,7 +26,9 @@ script:
   - echo "travis_fold:start:tx-pull"
   - ./scripts/ci/pull_translations.sh
   - echo "travis_fold:end:tx-pull"
-  - docker run -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e "ARCH=${ARCH}" -e "STOREPASS=${STOREPASS}" -e "KEYNAME=${KEYNAME}" -e "KEYPASS=${KEYPASS}" -e "VERSION=${TRAVIS_TAG}" opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
+  - APP_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "QField"; else echo "QField Beta ${TRAVIS_PULL_REQUEST}"; fi)
+  - PKG_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "qfield"; else echo "qfield_beta"; fi)
```

```suggestion
  - PKG_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "qfield"; else echo "qfield_beta_${TRAVIS_PULL_REQUEST}"; fi)
```

allow to have several apps?

---

I thought about that too.
I think it's usually enough to test one PR at a time and as a bonus you don't end up with 20 packages installed in parallel (and having to uninstall them all manually).

---

Fine with me, we can still fix it up later.